source/catalog.html

```diff
@@ -263,27 +305,23 @@ <h2 class="visually-hidden">Запостить своё фото.</h2>
                 <button type="button" class="editing-controls-list__button editing-controls-list__button--crop">
                   <span class="visually-hidden">Кадрирование.</span>
                   <svg class="editing-controls__icon" width="44" height="44" viewBox="0 0 44 44" fill="none" aria-hidden="true">
-                    <path d="M32 28H36V12C36 10.9391 35.5786 9.92172 34.8284 9.17157C34.0783 8.42143 33.0609 8 32 8H16V12H32V28ZM12 32V0H8V8H0V12H8V32C8 33.0609 8.42143 34.0783 9.17157 34.8284C9.92172 35.5786 10.9391 36 12 36H32V44H36V36H44V32H12Z" fill="#D22856"/>
+                    <use xmlns="http://www.w3.org/2000/svg" xlink:href="#crop-icon"></use>
```

Comment:
А это тут что делает ))
```suggestion
                    <use xlink:href="#crop-icon"></use>
```

Reply:
Много таких `use` еще:

https://github.com/htmlacademy-adaptive/1853291-pink-25/blob/f52f6a79467adb7b2a296f3ccba5054acb827dd8/source/catalog.html#L308
https://github.com/htmlacademy-adaptive/1853291-pink-25/blob/f52f6a79467adb7b2a296f3ccba5054acb827dd8/source/catalog.html#L316
https://github.com/htmlacademy-adaptive/1853291-pink-25/blob/f52f6a79467adb7b2a296f3ccba5054acb827dd8/source/catalog.html#L324
https://github.com/htmlacademy-adaptive/1853291-pink-25/blob/f52f6a79467adb7b2a296f3ccba5054acb827dd8/source/catalog.html#L374
https://github.com/htmlacademy-adaptive/1853291-pink-25/blob/f52f6a79467adb7b2a296f3ccba5054acb827dd8/source/catalog.html#L382
https://github.com/htmlacademy-adaptive/1853291-pink-25/blob/f52f6a79467adb7b2a296f3ccba5054acb827dd8/source/catalog.html#L390